packages/shared/src/contexts/PaymentContext.tsx

```diff
@@ -175,22 +197,8 @@ export const PaymentContextProvider = ({
         ({ appsId }) => appsId === PlusPriceTypeAppsId.EarlyAdopter,
       );
 
-      if (earlyAdopter?.value) {
-        return earlyAdopter.value;
-      }
-
-      const monthlyPrices = productOptions.filter(
-        (option) => planTypes[option.value] === PlusPriceType.Monthly,
-      );
-
-      if (monthlyPrices.length <= 1) {
-        return null;
-      }
-
-      return monthlyPrices.reduce((acc, plan) => {
-        return acc.priceUnformatted < plan.priceUnformatted ? acc : plan;
-      }).value;
-    }, [planTypes, productOptions]);
```

Comment:
Let's make sure that the "Custom Data -> appsId" in the Paddle side from the product team to have the id we have for `PlusPriceTypeAppsId.EarlyAdopter` since we did not remove this for backwards compat during the change.